Makefile

```diff
@@ -121,10 +121,11 @@ vet: gowork ## Run go vet against code.
 	go vet ./...
 	go vet ./... ./api/...
 
+APIPATH ?= $(shell pwd)/api
 .PHONY: tidy
 tidy: fmt
 	go mod tidy; \
-	pushd "$(LOCALBIN)/../api"; \
+	pushd $(APIPATH); \
```

Comment:
I think this is covered in #196. I'm also wondering why we can't just have something like:

```
.PHONY: tidy
tidy: fmt
	go mod tidy
	cd ./api
	go mod tidy
```

given we're already in the root directory of the operator.